src/Comet/EnvironmentAware.cs

```diff
@@ -232,7 +232,7 @@ public static T SetEnvironment<T>(this T contextualObject, Type type, string key
 			return contextualObject;
 		}
 
-		public static T SetEnvironment<T,TValue>(this T view, Type type, string key, Binding<TValue> binding, bool cascades = true, ControlState state = ControlState.Default)
+		public static T SetEnvironment<T, TValue>(this T view, Type type, string key, Binding<TValue> binding, bool cascades = true, ControlState state = ControlState.Default)
```

Comment:
In the future lets try and avoid white noise changes.

Reply:
It will be better to have some tools to format code in current code base.